projects/hipdnn/frontend/include/hipdnn_frontend/attributes/BatchnormAttributes.hpp

```diff
@@ -349,4 +347,3 @@ class BatchnormAttributes : public Attributes<BatchnormAttributes>
 
 typedef BatchnormAttributes Batchnorm_attributes;
 }
```

Comment:
```suggestion
} // namespace hipdnn_frontend::graph
```
Just pointing these out, but again nit.